docs/Principles/modeling-principles.md

```diff
@@ -0,0 +1,27 @@
+# Modeling Principles
+
+## Conforms to Schema
+The Monarch Biolink Specification is an implementation of the Biolink Model.
+The KG must be conformant with The Monarch Biolink Specification.
+
+## Node Normalization
+The final KG must have Nodes normalized to the canonical prefix for any given node type.
+The canonical prefix should be determined by The Monarch Biolink Model Specification.
+
+## Authoratative Source
+Providers of Associations are not the authoratative sources for the Nodes in general. Nodes should be ingested from their
+own authoratative source, seperate from edge ingests.
+
+## Genes and Proteins
+Genes and reference Proteins shall be treated as equivalent.
```

Comment:
Proteins -> proteins

Reply:
Let's keep the biolink classes capitalized (or better: linked)